sentry-android-core/src/main/java/io/sentry/android/core/AndroidOptionsInitializer.java

```diff
@@ -209,19 +209,21 @@ private static void installDefaultIntegrations(
       options.addIntegration(
           new ActivityLifecycleIntegration(
               (Application) context, buildInfoProvider, activityFramesTracker));
+      options.addIntegration(new CurrentActivityIntegration((Application) context));
       options.addIntegration(new UserInteractionIntegration((Application) context, loadClass));
       if (isFragmentAvailable) {
         options.addIntegration(new FragmentLifecycleIntegration((Application) context, true, true));
       }
-      options.addEventProcessor(
-          new ScreenshotEventProcessor((Application) context, options, buildInfoProvider));
+      options.addEventProcessor(new ScreenshotEventProcessor(options, buildInfoProvider));
```

Comment:
`m`: we could actually move the `ScreenshotEventProcessor` out of this if-condition and add it where we add the view hierarchy one.

Reply:
btw, can we actually move these two from this method, because it's called `installDefaultIntegrations` - maybe we could just move it to the `initializeIntegrationsAndProcessors` method, where we add other event processors?